raiden/tests/integration/fixtures/transport.py

```diff
@@ -63,6 +63,7 @@ def matrix_transports(
         server = local_matrix_servers[transport_index % len(local_matrix_servers)]
         transports.append(
             MatrixTransport(
+                bytearray(),  # must be set before starting the transport
```

Comment:
we could have a raiden services weaved into this fixture, and then extract the addresses from them so that they can be set in the matrix transport constructors.

the reason this is not done is because:
- for some tests, the cardinality between transport layers and raiden services is not 1:1.
- for some tests, transport layers are re-started on different raiden services, and so their addresses must be re-set.

this makes it—in my opinion—not worth the hassle to parametrize.

as a result, `_address` fields are set to their respective raiden service addresses right before starting the transport layers.

Reply:
Came to the same tought about passing raiden service addresses to the fixture. Agree with this.

Reply:
it's certainly not ideal, and i would love to iterate on these tests so that they can be re-structured. 

but i believe this is out of scope for this PR.